app/IATI/Services/ElementCompleteService.php

```diff
@@ -403,7 +406,7 @@ public function isPolicyMarkerElementCompleted($activity): bool
      * @param $activity
      *
      * @return bool
-     * @throws \JsonException
+     * @throws JsonException
```

Comment:
place all the throws before return and leave an empty line between them in this file